From 1f8b6fec6752d619b0ad2e856c34cd6b11d88c80 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Mon, 7 Jul 2014 08:47:37 +0200
Subject: ARM: shmobile: marzen: Consistently use tabs for indentation

Unify white space usage by consistently using tabs for indentation.

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 08af64097d4c5cb17f9df4dadf80ed5f508db743)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/boot/dts/r8a7779-marzen.dts | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7779-marzen.dts b/arch/arm/boot/dts/r8a7779-marzen.dts
index 20b176807848..5745555df943 100644
--- a/arch/arm/boot/dts/r8a7779-marzen.dts
+++ b/arch/arm/boot/dts/r8a7779-marzen.dts
@@ -112,17 +112,17 @@
 };
 
 &scif2 {
-       pinctrl-0 = <&scif2_pins>;
-       pinctrl-names = "default";
+	pinctrl-0 = <&scif2_pins>;
+	pinctrl-names = "default";
 
-       status = "okay";
+	status = "okay";
 };
 
 &scif4 {
-       pinctrl-0 = <&scif4_pins>;
-       pinctrl-names = "default";
+	pinctrl-0 = <&scif4_pins>;
+	pinctrl-names = "default";
 
-       status = "okay";
+	status = "okay";
 };
 
 &sdhi0 {
-- 
2.1.2

